src/script/components/AppContainer/AppContainer.tsx

```diff
@@ -49,6 +51,7 @@ export const AppContainer: FC<AppProps> = ({config, clientType}) => {
   window.wire.app = app;
   const mainView = new MainViewModel(app.repository);
   useTheme(() => app.repository.properties.getPreference(PROPERTIES_TYPE.INTERFACE.THEME));
+  useAccentColor();
```

Comment:
This will set the correct class to the `body` so that the modals are styled with the user's accent color